src/stream/src/cache/managed_lru.rs

```diff
@@ -113,38 +105,10 @@ impl<K: Hash + Eq + EstimateSize, V: EstimateSize, S: BuildHasher, A: Clone + Al
 
     /// Evict epochs lower than the watermark
     fn evict_by_epoch(&mut self, epoch: u64) {
-        let report = |lru: &Self, epoch: u64, evicted: usize| {
-            if evicted == 0 {
-                return;
-            }
-            lru.memory_evicted_metrics
-                .with_guarded_label_values(&[
-                    &lru.metrics_info.table_id,
-                    &lru.metrics_info.actor_id,
-                    &lru.evict_sequence.to_string(),
```

Comment:
I suppose `evict_sequence`, together with `epoch`, causes a large number of combinations/labels?

Reply:
It might be. But there is a guard around each metrics report. The new label will be immediately dropped after collected.

Reply:
But I'm not sure how it is implemented on the metrics server side or if it may cause a amplification.